solutions/ac-MadeUp.mdx

```diff
@@ -0,0 +1,133 @@
+---
+id: ac-MadeUp
+source: AtCoder Beginner Contest
+title: Made Up
+author: Kevin Sheng
+---
+
+[Official Editorial (C++)](https://atcoder.jp/contests/abc202/editorial/1894)
+
+Since $N$ can go all the way up to $10^5$, it would take much too long to iterate through
+all possible values of $(i, j)$.
+
+Instead, let's try and go through all possible values of $j$ and see how many values of $i$
+can be paired with it. To do this, we need to see how many numbers in $A$ are equal to $B_{C_j}$,
+which can be done with a map and some precalculation.
+
+<Info>
```

Comment:
```suggestion
<Info title="Official Solution">
```
maybe add a title so the info block doesn't look so weird?
(this title isn't that good, so if you can think of something better, you can replace it!)